modules/ROOT/partials/integrations/react-tech-ref.adoc

```diff
@@ -178,6 +181,28 @@ include::partial$misc/get-an-api-key.adoc[]
 />
 ----
 
+[[licenseKey]]
+=== `+licenseKey+`
+
+{cloudname} License key.
+
+Use this when self-hosting {productname} instead of loading from {cloudname}. For more information, see: xref:license-key.adoc[License Key].
+
+*Type:* `+String+`
+
+*Default value:* `+undefined+`
+
+*Possible values:* `undefined`, `'gpl'` or a valid {productname} license key
+
+==== Example: using `+licenseKey+`
+
+[source,jsx]
+----
+<Editor
+  licenseKey='your-license-key'
+/>
+----
```

Comment:
@danoaky-tiny Should we represent the `license_key` as the below to match [Setting the license_key](https://www.tiny.cloud/docs/tinymce/latest/license-key/)?
```suggestion
[source,jsx]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  license_key: 'gpl'
});
----
```

Reply:
The React integration overrides the `license_key` init prop on purpose with this component prop `licenseKey`. It's so it's consistent with how the hidden `api_key` editor is overriden by the `apiKey` component prop